src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 0; i < 6; i++) {
```

Comment:
to avoid magic number: 
```suggestion
        for (int i = 0; i < figures.length; i++) {
```